src/components/Card/Card.js

```diff
@@ -0,0 +1,63 @@
+import PropTypes from 'prop-types'
+import './card.css'
+
+function Card({ jobData }) {
+  let date = ''
+  try {
+    date = new Date(jobData.created).toLocaleDateString()
+  } catch (err) {
+    console.error('Invalid date.', jobData.created)
+  }
+
+  function removeHTML(text) {
+    // this regex removes html tags
+    return text.replace(/<[^>]*(>|$)|&nbsp;|&zwnj;|&raquo;|&laquo;|&bull;|&gt;/g, '')
+  }
+
+  const hasPhrase = (given, testStr) => given.toLowerCase().indexOf(testStr) > -1
+  function isRemote({ title, description }) {
+    return hasPhrase(description, 'remote') ||
```

Comment:
Have u tested this and does it work? I've had trouble with multiple line return statements usually I have to put () around the stuff to let JS know it's multi line. But if it works than I'm good with it.
4 me usually

```javascript
function () {
 return (
    /* 
       ... multiline blah
    */ 
  );
}
```

Reply:
It works. I think prettier did that honestly.

Reply:
We will try it with a parentheses and and if else block and see if that's kosher￼

Reply:
this is now removed